### PR TITLE
Added new awesome person to describe cmd

### PR DESCRIPTION
**Pull request by opdelta**
:)

### CODE REVIEW

**Comment by AlexandreLachanceGit** [comment]
I don't approve. We have to stay accurate sorry.

**Review by AlexandreLachanceGit — CHANGES_REQUESTED** [review]
Put my id instead.

File: src/functions/describe/main.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def describe(request):
 
         # Special user are protected. This will go with firebase in the future
         if mentions:
-            if str(mentions[0]) == "135048445097410560" or str(mentions[0]) == "964693548396068916":
+            if str(mentions[0]) == "135048445097410560" or str(mentions[0]) == "964693548396068916" or str(mentions[0]) == "127947460462116864":
```
Comment (ArmandBriere) [inline src/functions/describe/main.py:21]: Max line length is 120, run `tox`

**Review by ArmandBriere — CHANGES_REQUESTED** [review]

**Comment by opdelta** [comment]
> Put my id instead.

If I put yours and mine and you approve, we got a deal

File: src/functions/describe/main.py (hunk 1 above)
Comment (opdelta) [reply src/functions/describe/main.py]: Changes made in latest commit :)

**Review by ArmandBriere — CHANGES_REQUESTED** [review]
Permission denied (Contact admin for more information)

**Pull request closed without merging**